modules/aleph/includes/aleph.availability.inc

```diff
@@ -78,8 +78,9 @@ function aleph_availability_holdings(array $provider_ids) {
         $sub_libraries[$item->getSubLibraryCode()]['reservable'] = $item->isReservable();
       }
       else {
+        $placements = implode(' > ', array_filter($item->getPlacements()));
```

Comment:
A single comment about why the ' > ' implode is done would be nice. Is this to potentially show multiple locations, or is it a kind of "breadcrumb"?